### ISSUE TITLE
Files 2.0 Features Enhancements

### ISSUE THREAD

**Issue by miguelisidoro** [files-community/Files#8316]
### What's the Problem?

Hello,

I am now using Files 2.0 and I am liking it but still am running in a few issues.

How can I open each folder in a new window instead of a new tab?

How can I display the folder full path instead of the current folder name?

These are features available in Windows 10 File Explorer and Windows 11 File Explorer (that has slow navigation issues with Cloud Drives).

Also, the right mouse click menu with a full context menu takes a few seconds to show the complete full context menu. Any way to speed up this process?

Thanks

### Solution/Idea

1 - Add options to open Files in a new Windows instead of tabs
2 - Add option to display folder full path as in Windows 10 File Explorer
3 - Speed up the process of showing the full context menu to be immediate as in Windows 10 File Explorer

### Alternatives

Not applicable

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

### Files Version

2.1.13.0

### Windows Version

Windows 11 21H2

**Comment by Josh-65** [comment]
The full path to a folder is already shown in the path bar.

**Comment by miguelisidoro** [comment]
> 

This is true but I would like it to be available in the taskbar also and in ALT+Tab so that I can easily navigate to the folder I want.
This is available in Windows Explorer already (option "Display the full path in the title bar" - https://www.onmsft.com/how-to/how-to-make-windows-10s-file-explorer-show-the-full-directory-path-in-the-title-bar

Thanks

**Comment by yair100** [comment]
@miguelisidoro we believe the performance issues are resolved in the latest Files Preview build. Can you please try it out and let us know your thoughts? https://files.community/appinstallers/Files.preview.appinstaller
